integration-test/src/main/java/com/sequenceiq/it/cloudbreak/cloud/v4/yarn/YarnCloudProvider.java

```diff
@@ -61,6 +62,29 @@ public CloudPlatform getCloudPlatform() {
         return CloudPlatform.YARN;
     }
 
+    @Override
+    public VerticalScalingTestDto freeIpaVerticalScalingTestDto(VerticalScalingTestDto verticalScalingTestDto) {
+        return verticalScalingTestDto.withGroup(yarnProperties.getVerticalScale().getFreeipa().getGroup())
```

Comment:
As I see all of the Cloud providers. could you please check these lines as well?